packages/node_modules/@webex/webex-core/test/unit/spec/credentials/credentials.js

```diff
@@ -228,6 +228,29 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#extractOrgIdFromUserToken()', () => {
+      let credentials;
+      let webex;
+
+      beforeEach('generate webex and destructure credentials', () => {
+        webex = new MockWebex();
+        credentials = new Credentials(undefined, {parent: webex});
+      });
+
+      it('should return the OrgId of the provided user token', () => {
+        const userToken = 'ZjY4NmE4YjgtZmNhMC00MTllLTllNmEtM2M2OGU4ZDg2MzFkMTJkNjVlZjMtYTc0_A52D_584cf4cd-eea7-4c8c-83ee-67d88fc6eab5';
```

Comment:
This was not a valid token was it?

Reply:
This was an old integration test user token that has expired.

Reply:
Might be beneficial to change the token a bit to one that was never issued anywhere. Just as a safeguard.